lib/rucio/db/sqla/util.py

```diff
@@ -302,3 +302,17 @@ def json_implemented(session=None):
         return False
 
     return True
+
+
+def try_drop_constraint(constraint_name, table_name):
+    """
+    Tries to drop the given constrained and returns successfully if the
+    constraint already existed on Oracle databases.
+
+    :param constraint_name: the constraint's name
+    :param table_name: the table name where the constraint resides
+    """
+    try:
+        op.drop_constraint(constraint_name, table_name)
```

Comment:
does this work without explicitly specifying the constraint type? previously this was required to add

Reply:
that is only necessary for mysql, according to the documentation https://alembic.sqlalchemy.org/en/latest/ops.html?highlight=drop_constraint#alembic.operations.Operations.drop_constraint (and confirmed by the tests)